package.json

```diff
@@ -12,35 +12,38 @@
     "test": "jest",
     "cli": "nodemon src/cli/load-license-codes.ts"
   },
+  "engines": {
+    "node": ">=18.0.0"
+  },
   "devDependencies": {
-    "@internxt/eslint-config-internxt": "^1.0.8",
+    "@internxt/eslint-config-internxt": "^1.0.9",
     "@internxt/prettier-config": "^1.0.2",
-    "@types/jest": "^27.5.1",
-    "@types/jsonwebtoken": "^8.5.8",
-    "@types/node": "^17.0.35",
-    "@typescript-eslint/eslint-plugin": "^5.26.0",
-    "@typescript-eslint/parser": "^5.26.0",
-    "eslint": "^8.16.0",
-    "jest": "27",
-    "nodemon": "^2.0.16",
-    "pino-pretty": "^7.6.1",
-    "prettier": "^2.6.2",
-    "ts-jest": "27",
-    "ts-node": "^10.8.0",
-    "typescript": "^4.7.2",
+    "@types/jest": "^29.5.14",
+    "@types/jsonwebtoken": "^9.0.7",
+    "@types/node": "^22.9.0",
+    "@typescript-eslint/eslint-plugin": "^8.14.0",
+    "@typescript-eslint/parser": "^8.14.0",
+    "eslint": "^9.14.0",
+    "jest": "29.7.0",
+    "nodemon": "^3.1.7",
+    "pino-pretty": "^13.0.0",
+    "prettier": "^3.3.3",
+    "ts-jest": "29.2.5",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.6.3",
     "xlsx": "^0.18.5"
   },
   "dependencies": {
-    "@fastify/cors": "^7.0.0",
-    "@fastify/jwt": "^5.0.1",
-    "axios": "^0.27.2",
-    "dayjs": "^1.11.12",
-    "dotenv": "^16.0.1",
-    "fastify": "^3.29.0",
-    "fastify-rate-limit": "^5.9.0",
-    "ioredis": "^5.0.6",
-    "jsonwebtoken": "^8.5.1",
-    "mongodb": "^4.6.0",
+    "@fastify/cors": "^10.0.1",
+    "@fastify/jwt": "^9.0.1",
+    "@fastify/rate-limit": "^10.1.1",
+    "axios": "^1.7.7",
+    "dayjs": "^1.11.13",
+    "dotenv": "^16.4.5",
+    "fastify": "^5.1.0",
+    "ioredis": "^5.4.1",
+    "jsonwebtoken": "^9.0.2",
+    "mongodb": "^6.10.0",
```

Comment:
Does this version match with the current MongoDB engine we have (Mongo 6.0)? It's important to check so to avoid incompatibilities in the queries this server runs. You can test it using the virtualized MongoDB 6.0 image from DockerHub locally.